python/src/RatioOfUniforms_doc.i.in

```diff
@@ -0,0 +1,487 @@
+%feature("docstring") OT::RatioOfUniforms
+"Ratio of Uniforms simulation algorithm.
+
+
+.. warning::
+    This class is experimental and likely to be modified in future releases.
+    To use it, import the ``openturns.experimental`` submodule.
+
+Available constructors:
+    RatioOfUniforms(*dist*)
+
+    RatioOfUniforms(*logUnscaledF, rangeF, isScaled*)
+    
+Parameters
+----------
+dist : :class:`~openturns.Distribution`
+    Distribution to sample.
+    
+    With that constructor, the range is the distribution range and *isScaled* = True.
+
+logUnscaledPDF : :class:`~openturns.Function`
+   Function :math:`\log F` such that :math:`f = F\, 1_{I}`  is proportional to the PDF :math:`p` we want to sample.
```

Comment:
In the rest of the documentation (see for instance Subset or NAIS) the indicator function is in "bold":  \mathbf{1}

Reply:
yes. I modified the text.